modules/jupyter_utils.ipynb

```diff
@@ -170,9 +172,9 @@
     "\n",
     "\n",
     "def _meandice_transform(x):\n",
-    "    y_pred = x[CommonKeys.PRED]\n",
-    "    y_pred = predict_segmentation(y_pred)\n",
-    "    return y_pred, x[CommonKeys.LABEL]\n",
+    "    y_pred = [predict_segmentation(i[CommonKeys.PRED]) for i in x]\n",
```

Comment:
Should use `decollate_batch()` instead of `for i in x`?

Reply:
Hi Nic, this line is used in metric calculation step, which is after the decollate step in workflow.